src/main/kotlin/me/zeroeightsix/kami/feature/module/MurderMysteries.kt

```diff
@@ -0,0 +1,73 @@
+package me.zeroeightsix.kami.feature.module
+
+import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting
+import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting.Constrain
+import me.zero.alpine.listener.EventHandler
+import me.zero.alpine.listener.Listener
+import me.zeroeightsix.kami.event.TickEvent
+import me.zeroeightsix.kami.util.text
+import net.minecraft.item.*
+import net.minecraft.util.Formatting
+
+@Module.Info(
+        name = "MurderMysteries",
+        category = Module.Category.MISC,
+        description = "Get notified when a player holds a dangerous item"
+)
+object MurderMysteries : Module() {
+
+    @Setting(name = "InnocuousItems")
+    private var innocuousItems = true
+    @Setting(name = "WeaponItems")
+    private var weaponItems = true
+    @Setting(name = "ToolItems")
+    private var toolItems = true
+    @Setting(name = "DetectiveItems")
+    private var detectiveItems = false
+    @Setting(name = "IgnoreWoodenShovel")
+    private var ignoreWoodenShovel = false
+    @Setting(name = "CooldownOnOutput") // This is how often it will tell the player in chat
+    private var updateEvery: @Constrain.Range(min = 2.0, max = 16.0, step = Double.MIN_VALUE) Int = 10
+
+    private var lastUpdate: Long = 0L
+
+    @EventHandler
+    val worldListener = Listener<TickEvent.Client.InGame>({
+        if ((lastUpdate + updateEvery * 1000L <= System.currentTimeMillis()))
+            mc.world?.players?.forEach { player ->
+                //Make sure the player is not being told that they are the murderer
+                if (player == mc.player) return@forEach
+
+                val innocentItemsList = listOf(
+                        Items.CARROT,
+                        Items.BONE,
+                        Items.CARROT_ON_A_STICK,
+                        Items.GOLDEN_CARROT,
+                        Items.BLAZE_ROD,
+                        Items.SPONGE
+                )
+                val detectiveItemsList = listOf(
+                        Items.BOW,
+                        Items.SNOWBALL,
+                        Items.ARROW
+                )
```

Comment:
Move the declaration of these lists outside of the tick listener. No need to initialise them every tick if we're not going to change them anyways.